testproject/testproject/settings.py

```diff
@@ -105,9 +105,9 @@
     'wiki',
     'wiki.plugins.help',
     'wiki.plugins.links',
-    'wiki.plugins.images',
-    'wiki.plugins.attachments',
-    'wiki.plugins.notifications',
+    'wiki.apps.ImagesConfig',
```

Comment:
https://github.com/django-wiki/django-wiki/blob/main/testproject/testproject/settings/base.py#L44
matching from the original repo.

Reply:
Otherwise it fails to run migration in django42.